### PR TITLE
Deleting files related to streak reset (No longer in use). #83

### PR DESCRIPTION
**Pull request by SirKanaad26**
We have gotten rid of the streak reset component. Removing it.

### CODE REVIEW

File: src/__tests__/scripts/task_page.test.js
Comment (Earth34r) [inline src/__tests__/scripts/task_page.test.js:1]: Does this file need to be deleted? It contains useful tests for the task page, no?

**Review by lezhangleonard — APPROVED** [review]
Test it locally, works well!

File: src/styles/pages/task_page.css
Comment (Earth34r) [inline src/styles/pages/task_page.css:1]: Is this file being replaced or moved?

Comment (SirKanaad26) [reply src/styles/pages/task_page.css:1]: Removed

File: src/__tests__/scripts/task_page.test.js
Comment (SirKanaad26) [reply src/__tests__/scripts/task_page.test.js:1]: No, there's a separate file for the tasks page called __tests__/pages/task-page.html.test.js

Comment (Earth34r) [reply src/__tests__/scripts/task_page.test.js:1]: That tests the html file, as far as I know. The task_page.test.js tests the file task_page.js, and has valid tests. Maybe we should move them to the .html.test?

**Review by Earth34r — CHANGES_REQUESTED** [review]
src/__tests__/scripts/task_page.test.js contains valid tests for the file task_page.test.js, please rework these so they are included in a different file.

File: src/__tests__/scripts/task_page.test.js
Comment (SirKanaad26) [reply src/__tests__/scripts/task_page.test.js:1]: No, those tests were not for the task-page, they were for the task_page file. The task_page file wasn't being used in our project. Kindly check functionality and see if it works in local and then review the code.

**Review by commcheck396 — APPROVED** [review]

**Review by Earth34r — APPROVED** [review]
Cleared up which additional files are being deleted, thanks! Looks good!

**Pull request merged**